guide/src/examples/hello-world.md

```diff
@@ -38,7 +38,7 @@ Our JS entry point is quite small!
 
 ## Webpack-specific files
 
-> **Note**: Webpack is not required for this example, and if you're interested
+> **Note**: Webpack is required for this example, and if you're interested
```

Comment:
Can you please remove the entire note, since it doesn't make sense to keep it here?

Reply:
Why? I found it interesting